.travis.yml

```diff
@@ -211,3 +211,12 @@ matrix:
         submodules: false # avoid cloning ethereum/tests
       script:
         - go run build/ci.go purge -store gethstore/builds -days 14
+
+    - name: Race Detector for Swarm
+      if: ethersphere/go-ethereum AND type = cron
+      os: linux
+      dist: trusty
+      go: 1.11.x
+      git:
+        submodules: false # avoid cloning ethereum/tests
+      script: ./build/travis_keepalive.sh go test -v -timeout 20m -race ./swarm...
```

Comment:
I'd be inclined to also run the race detector on `p2p` and `cmd/swarm`

Reply:
I like the idea. I created a [ticket](https://github.com/ethersphere/go-ethereum/issues/1249), as now I'm confident the build is stable and I want to have a vacation soon. ;)